core/src/main/java/org/itsallcode/openfasttrace/core/cli/CliStarter.java

```diff
@@ -128,7 +127,11 @@ public void run()
         case TraceCommand.COMMAND_NAME:
             performable = new TraceCommand(this.arguments);
             break;
+        case HelpCommand.COMMAND_NAME:
+            performable = new HelpCommand();
+            break;
         default:
+            new HelpCommand().run();
             throw new IllegalStateException(
                     "Unknown command '" + command.get() + "' trying to execute OFT mode.");
```

Comment:
This is error not necessary anymore if the default action is showing the help. But the exit status should still be negative in case this is used in a build by accident.

Reply:
✅